### ISSUE TITLE
help

### ISSUE THREAD

**Issue by trinityy66** [EEEGUI/SceneParsingNetwork#1]
Can I borrow the reproduced TRL code for reference? The accuracy of my reproduction is low.I use tensorflow but I'm new in pytorch .This is my email：yujin@mail.dlut.edu.cn. Thank you.
